static/books/unix.html

```diff
@@ -186,7 +186,7 @@
 </P>
 
 <P ALIGN=JUSTIFY>
-AT&amp;T had actually been rather forthcoming with licenses to Unix for academic use.  According to <a href="http://www.admin.com">The Cranberry Book</a>, Unix version 6 (circa 1976) was free for Universities and version 7 cost $100 (government labs and commercial entities had to pay $21,000).  This was partly due to a consent decree that forbade AT&amp;T from selling software commercially.
+AT&amp;T had actually been rather forthcoming with licenses to Unix for academic use.  According to <a href="https://web.archive.org/web/19961023230807/http://www.admin.com/">The Cranberry Book</a>, Unix version 6 (circa 1976) was free for Universities and version 7 cost $100 (government labs and commercial entities had to pay $21,000).  This was partly due to a consent decree that forbade AT&amp;T from selling software commercially.
```

Comment:
On Tue, Aug 23, 2016 at 12:35:30PM -0700, Remy DeCausemaker wrote:
> In static/books/unix.html:
> 
> > @@ -186,7 +186,7 @@
> >  </P>
> >
> >  <P ALIGN=JUSTIFY>
> > -AT&amp;T had actually been rather forthcoming with licenses to Unix for academic use.  According to <a href="http://www.admin.com">The Cranberry Book</a>, Unix version 6 (circa 1976) was free for Universities and version 7 cost $100 (government labs and commercial entities had to pay $21,000).  This was partly due to a consent decree that forbade AT&amp;T from selling software commercially.
> > +AT&amp;T had actually been rather forthcoming with licenses to Unix for academic use.  According to <a href="https://web.archive.org/web/19961023230807/http://www.admin.com/">The Cranberry Book</a>, Unix version 6 (circa 1976) was free for Universities and version 7 cost $100 (government labs and commercial entities had to pay $21,000).  This was partly due to a consent decree that forbade AT&amp;T from selling software commercially.
> 
> still busted link prolly

Works for me, so I don't know, might just be archive.org is, as with so many
good things, underprovisioned and thus slow? Or have I screwed up the HTML
somehow that I'm overlooking?

There are pages for the 4th edition (no longer with a red cover) at the
usual booksellers, but a link to one of those doesn't seem to be in the
spirit of what Nick is about.

I'd be tempted to drop this in if the archive.org link proves continually to
be untenable:

https://en.wikipedia.org/wiki/Evi_Nemeth

-- 
Joe